mantidproject/mantid
# Add a PropertyManagerProperty type

Adds a `PropertyManagerProperty` to allow a `PropertyManager` as a value in another `PropertyManager` without having to create a name and put it in the `PropertyManagerDataService`

**To test:**

There is limited external testing that can be done with this. Review the code and check you agree with how the property type is handled. The ultimate goal is to allow automatic conversion of a Python `dict` to a `PropertyManager` on input to an algorithm.

Fixes #16028

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [ ] If there are changes in the release notes then do they describe the changes appropriately? -> N/A
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed? ->N/A
- [ ] How do the changes handle unexpected situations, e.g. bad input? ->N/A
- [ ] Has the relevant documentation been added/updated? ->N/A
- [ ] Is user-facing documentation written in a user-friendly manner? ->N/A
- [ ] Has developer documentation been updated if required? ->N/A
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
